sympy/functions/elementary/complexes.py

```diff
@@ -732,6 +732,18 @@ class arg(Function):
 
     @classmethod
     def eval(cls, arg):
+        a = arg
+        for i in range(3):
+            if isinstance(a, cls):
+                a = a.args[0]
+            else:
+                break
+        else:
+            # arg(x) is real: -ve | 0 | +ve
```

Comment:
It might be helpful to (also) put this in the docstring/doctest. A user might be surprised to see that their nested `arg`s denest to a `S.NaN` and shouldn't have to read the source to get the explanation.